### DISCUSSION TITLE
When will Pydantic v1 no longer be maintained?

### DISCUSSION THREAD

**Discussion by mattwelborn** [pydantic/pydantic#7505] [Question] [answered]
Do you have plans to EOL Pydantic v1? This could include ending new 1.x releases, or ending support for new Python versions for v1.

**Comment by hramezani** [comment, marked as answer]
`1.10` would be the last minor release on `V1` and it only gets patch releases. So, no new features will be available on `V1`.

Also, support for the new Python version will not be added to `V1`.

We will probably stop supporting `V1` after we released `V3`
